tests/app/views/test_frameworks.py

```diff
@@ -453,54 +453,61 @@ def test_stats(self):
 
         response = self.client.get('/frameworks/g-cloud-7/stats')
-        assert_equal(json.loads(response.get_data()), {
-            u'services': [
-                {u'count': 1, u'status': u'not-submitted',
-                 u'declaration_made': False, u'lot': u'iaas'},
-                {u'count': 2, u'status': u'not-submitted',
-                 u'declaration_made': True, u'lot': u'iaas'},
-                {u'count': 2, u'status': u'not-submitted',
-                 u'declaration_made': False, u'lot': u'paas'},
-                {u'count': 2, u'status': u'not-submitted',
-                 u'declaration_made': True, u'lot': u'paas'},
-                {u'count': 3, u'status': u'not-submitted',
-                 u'declaration_made': False, u'lot': u'saas'},
-                {u'count': 2, u'status': u'not-submitted',
-                 u'declaration_made': True, u'lot': u'saas'},
-                {u'count': 1, u'status': u'not-submitted',
-                 u'declaration_made': True, u'lot': u'scs'},
-
-                {u'count': 2, u'status': u'submitted',
-                 u'declaration_made': False, u'lot': u'iaas'},
-                {u'count': 4, u'status': u'submitted',
-                 u'declaration_made': True, u'lot': u'iaas'},
-                {u'count': 4, u'status': u'submitted',
-                 u'declaration_made': False, u'lot': u'paas'},
-                {u'count': 4, u'status': u'submitted',
-                 u'declaration_made': True, u'lot': u'paas'},
-                {u'count': 4, u'status': u'submitted',
-                 u'declaration_made': False, u'lot': u'saas'},
-                {u'count': 4, u'status': u'submitted',
-                 u'declaration_made': True, u'lot': u'saas'},
-                {u'count': 1, u'status': u'submitted',
-                 u'declaration_made': False, u'lot': u'scs'},
-                {u'count': 3, u'status': u'submitted',
-                 u'declaration_made': True, u'lot': u'scs'},
-            ],
-            u'interested_suppliers': [
-                {u'count': 7, u'declaration_status': None, u'has_completed_services': False},
-                {u'count': 1, u'declaration_status': None, u'has_completed_services': True},
-                {u'count': 5, u'declaration_status': 'complete', u'has_completed_services': False},
-                {u'count': 1, u'declaration_status': 'complete', u'has_completed_services': True},
-                {u'count': 4, u'declaration_status': 'started', u'has_completed_services': False},
-                {u'count': 2, u'declaration_status': 'started', u'has_completed_services': True},
-            ],
-            u'supplier_users': [
-                {u'count': 4, u'recent_login': False},
-                {u'count': 2, u'recent_login': None},
-                {u'count': 5, u'recent_login': True},
-            ]
-        })
+        json_response = json.loads(response.get_data())
+
+        services = json_response[u'services']
```

Comment:
When the test failed, all I got was a JSON blob marked as different, and it was hard to tell which bit had changed without copying and pasting somewhere to do a diff, and even then diffing JSON is a bit dubious... so checking each separate bit, gives a much clearer indication of which bit of code we are test / is broken. In the successful case it's no different, in the failure case you get easier debugging / more detail.